.env

```diff
@@ -2,22 +2,22 @@
 VERBOSE_HARNESS=0
 
 # Used to determine sandbox build type:
-TYPE="channel" # OR "source"
+TYPE="source" # OR "source"
```

Comment:
TODO: Change back to `channel`

Reply:
After `feature/box-storage` is merged in go-algorand:
```suggestion
TYPE="channel" # OR "source"
```